### DISCUSSION TITLE
Output Capstone Project Flask UI

### DISCUSSION THREAD

**Discussion by fxpeterbs31** [teamalgoritma/community#179] [Q&A] [answered]
Halo, saya mau bertanya. Apakah output pada task 5 pada codingan kita harus sama dengan output yang terletak pada cell dibawahnya?? dalam kasus ini pada top_category.head() yang diatas apakah harus sama dengan yang bawah?
<img width="263" alt="Screenshot 2022-07-08 010202" src="https://user-images.githubusercontent.com/107398327/177839818-7083c75d-2112-4321-bae5-7533280c99bd.png">

**Comment by amaliasf** [comment, marked as answer]
Halo Pak Peter, benar Pak seharusnya hasil tabel agregasi-nya sama. Output tersebut sudah disesuaikan dengan nilai yang sebenarnya dari data.
Bisa dipastikan lagi Pak apakah Bapak sudah melakukan cleaning terhadap data yang ter-duplicate. Semoga membantu Pak :)
